packages/camera/camera_windows/windows/camera_plugin.cpp

```diff
@@ -398,8 +401,20 @@ void CameraPlugin::CreateMethodHandler(
       resolution_preset = ResolutionPreset::kAuto;
     }
 
+    const auto* fps_argument = std::get_if<int>(ValueOrNull(args, kFpsKey));
+    int fps = fps_argument ? *fps_argument : -1;
```

Comment:
I assume the `-1`s here are intended to be special sentinel values indicating unset? If so, they should either have special constants (`kUnsetFps`, `kUnsetBitrate`, etc.), or should use `std::optional` instead.

Reply:
refactored using `std::optional`